eng/pipelines/templates/stages/cosmos-sdk-client.yml

```diff
@@ -93,7 +93,7 @@ extends:
               SECONDARY_ACCOUNT_KEY: 'C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw=='
               # Increased timeout to 90 because of cosmos emulator taking 25-30 mins to download emulator
               # Issue filed to improve download speed: https://github.com/Azure/azure-sdk-for-java/issues/12970
-              TimeoutInMinutes: 90
+              TimeoutInMinutes: 120
```

Comment:
Hmm - this one is interesting. I have only noticed this recently - `Build for non-From Source run` fluctuates from 13 min - 16 min (which I thought is one of the contributors). I am only worried this will become lost technical debt.

Reply:
Agreed - pipeline run duration is too high anway right now - I would like to get to below 60 minutes. But this has to wait a few more weeks before anyone finds time for investigating it I guess.